### PR TITLE
MM-12755 Add reply mention highlight for replyBar

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Add reply mention highlight to replyBar

#### Ticket Link
[MM-12755](https://mattermost.atlassian.net/browse/MM-12755)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Has UI changes
- [x] has redux changes(https://github.com/mattermost/mattermost-redux/pull/713)

#### Device Information
This PR was tested on: [Emulators]

#### Screenshots
screenshot 2018-11-20 at 12 45 15 pm

### CODE REVIEW

File: app/components/post/post.js [hunk 1]
```diff
@@ -407,6 +413,10 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
         replyBarLast: {
             paddingBottom: 10,
         },
+        commentMentionBgColor: {
+            backgroundColor: theme.mentionHighlightBg || 'rgb(255, 255, 0)',
```
Comment (enahum) [inline app/components/post/post.js]: Why the or here?

Comment (sudheerDev) [reply app/components/post/post.js]: > Why the or here?

I didn't quite get that.

we are picking the color from theme if exits and defaulting to yellow as we do in webapp.
If you are asking about why even a style. I am 0/5 and just following other how we are adding other condition classes for replyBar.

Comment (enahum) [reply app/components/post/post.js]: I was asking about the ‘||’ that is an “or” syntax right? Hehehe

Anyway I don’t think we need to default to that and the webapp code probably to old

Comment (sudheerDev) [reply app/components/post/post.js]: > I was asking about the ‘||’ that is an “or” syntax right?

Aah got it!.

**Pull request merged**